README.md

```diff
@@ -42,13 +42,32 @@ Additionally, steps must have material and product rules which define the files
 
 *Hint: Take a look at [`create_layout.py`](https://github.com/in-toto/in-toto/blob/develop/demo/owner_alice/create_layout.py), a script that creates the in-toto demo layout.*
 
-#### Rules
-*TODO*
+#### Artifact Rules
+A software supply chain usually operates on a set of files, such as source code, executables, packages, or the like. in-toto calls these files artifacts or more precisely materials and products.
+A material describes the state of an artifact before a step or inspection has been carried out and a product describes its state thereafter.
+
+The in-toto layout provides a simple rule language to authorize or enforce the artifacts that should be used as materials for or result from the execution of a step or inspection and to link subsequent steps and inspections together. This adds the following guarantees for any given step or inspection:
+- Only artifacts **authorized** by the project owner are created, modified or deleted,
+- each defined creation, modification or deletion is **enforced**, and also
+- restricted to the scope of its definition, which **links** subsequent steps and inspections together.
+
+Note that it is up to you to provide the desired security guarantees, by authorizing, enforcing and linking material and product together using any and usually multiple of the following rules:
```

Comment:
I would rephrase:

> The desired security guarantees

to

> Properly secure your supply chain

Or something of the like. I don't think it's worth using the term security guarantees.